client/bennys.lua

```diff
@@ -5,9 +5,9 @@ RegisterNUICallback('bennys/getitems', function(_, cb)
     for _, v in pairs(Config.Bennys.Items) do
 
         translated[#translated + 1] = {
-            name = v.name,
-            label = QBCore.Shared.Items[v.name].label,
-            image = Config.Inventory .. "/html/images/" .. QBCore.Shared.Items[v.name].image,
+             name = v.item,

```

Comment:
item is not defined? In the `Config.Bennys.Items` its labeled as name not too sure why this is changed?

Reply:
v.name isn't defined but v.item is

Reply:
> item is not defined? In the `Config.Bennys.Items` its labeled as name not too sure why this is changed?

mb i ported the config from jl carboost 😅